File: 8kyu-Simple-Multiplication.js
// This kata is about multiplying a given number by eight if it is an even number and by nine otherwise.

const chai = require("chai");
const assert = chai.assert;
chai.config.truncateThreshold=0;

describe("Basic Tests", () => {
  it("Testing for fixed tests", () => {
    assert.strictEqual(simpleMultiplication(2),16,'Should return double given argument..')
    assert.strictEqual(simpleMultiplication(1),9,'Should return double given argument..')
    assert.strictEqual(simpleMultiplication(8),64,'Should return given argument times eight...')
    assert.strictEqual(simpleMultiplication(4),32,'Should return given argument times eight...')
    assert.strictEqual(simpleMultiplication(5),45,'Should return given argument times nine...')
  });
});


// my code


function simpleMultiplication(number) {
    return (number%2==0) ? number*8:number*9;
}

File: CodeWars/.cs/8kyu-SimpleMultiplication.cs
// This kata is about multiplying a given number by eight if it is an even number and by nine otherwise.

namespace Solution 
{
  using NUnit.Framework;

  [TestFixture]
  public class SolutionTest
  {
    [Test]
    public void BasicTests()
    {
      Assert.AreEqual(9, Multiplier.Multiply(1),"Should return given argument times nine");
      Assert.AreEqual(16, Multiplier.Multiply(2), "Should return given argument times eight");
      Assert.AreEqual(64, Multiplier.Multiply(8), "Should return given argument times eight");
      Assert.AreEqual(32, Multiplier.Multiply(4), "Should return given argument times eight");
      Assert.AreEqual(45, Multiplier.Multiply(5), "Should return given argument times nine");
    }
  }
}

// my code

public class Multiplier
{
  public static int Multiply(int x) 
  {
    return (x % 2 == 0) ? x * 8 : x * 9;
  }
}  